Index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <link rel="stylesheet" href="style.css"/>
+    <title>Mobile First Hussen Portfolio</title>
+</head>
+<body>
+<div class="main-container">
+    <div class="top-bar">
+        <div class="tool-bar">
+            <div class="logo-setting">
+                <div class="logo">
+                    <img src="img/My-Logo.png" alt="HY">
+                    <!-- <img src="img/Hu-Photo-2-3.jpeg" alt="HY" width="76px" height="20px"> -->
+                </div>
+            </div>
+            <div class="dash-setting">
+                <div class="icons">
+                        <div class="icon-Union">
+                            <img src="img/Union.png" alt="Three Dashs" >
+                        </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="content-section">
+        <div class="bg-shapes">
+            <div class="text-1">
+                 <h2>I’m Hussen Glad to see you!</h2>
+            </div>
+            <div class="text-2">
+                <p>I’m a software developer! I can help you build a product , 
+                    feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </div>
+                <!-- <div class="social-media-setting"> -->
+                    <!-- <div class="social-media-title"> -->
+            <div class="lets-connect">
+                <b>LET’S CONNECT</b>
+            </div>
+                    <!-- </div> -->
+                <!-- </div> -->
+            <div class="social-media-icons">
+                    <img src="img/social-media-icons.png" alt="social-media-icons" >
+            </div>
+
+        </div>
+    </div>
```

Comment:
- It will be best if you replaced using div in this section with a section tag, something like this 👉🏽 `<section class="content-section">` .  Because the container of this section should be a section tag, It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.